packages/e2e-tests/plugins/interactive-blocks/directive-context/render.php

```diff
@@ -134,3 +134,10 @@
 	<button data-testid="navigate" data-wp-on--click="actions.navigate">Navigate</button>
 	<button data-testid="async navigate" data-wp-on--click="actions.asyncNavigate">Async Navigate</button>
 </div>
+
+<div
+	data-wp-interactive='{"namespace": "directive-context-non-default"}'
+	data-wp-context--non-default='{ "text": "non default" }'
```

Comment:
I would also add a default context to check cases where two `wp-context` directives are set together.
```suggestion
    data-wp-context='{ "text": "default" }'
	data-wp-context--non-default='{ "text": "non default" }'
```

That needs adding another `<span>` below, consuming the default context.